windows/9.3.0/sitecore-assets/tools/entrypoints/worker/Development.ps1

```diff
@@ -0,0 +1,33 @@
+# setup
+$ErrorActionPreference = "STOP"
+
+# print start message
+Write-Host ("### Sitecore Development ENTRYPOINT, starting...")
+
+# check to see if we should start the Watch-Directory.ps1 script
+$useWatchDirectory = (Test-Path -Path "C:\src" -PathType "Container") -eq $true
+
+if ($useWatchDirectory)
+{
+    # start Watch-Directory.ps1 in background, kill foreground process if it fails
+    Start-Job -Name "WatchDirectory.ps1" {
+        try
+        {
+            # TODO: Handle additional Watch-Directory params, use param splattering?
+
+            & "C:\tools\scripts\Watch-Directory.ps1" -Path "C:\src" -Destination "C:\worker" -ExcludeFiles "Web.config"
+        }
+        finally
+        {
+            Get-Process -Name "filebeat" | Stop-Process -Force
```

Comment:
Since filebeat is not (yet) included here you can remove the try/finally

Reply:
Take a look @pbering